gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/model/GbUser.java

```diff
@@ -91,15 +91,24 @@ public boolean isValid() {
 		return StringUtils.isNotBlank(userUuid);
 	}
 
+	public String getSortName() {
```

Comment:
There is a GbUser constructor that takes a User object already. Rather than reimplementing the sortname logic here, you could call u.getSortName() in the constructor that takes the User object and store it. For the other constructor, the sort name could be passed in (and any call sites fixed to pass it in appropriately).

Reply:
I thought that was an entity on db, but I was wrong. This approach is much better! 
Thanks!!